ops/dev3/persistent/main.tf

```diff
@@ -10,7 +10,7 @@ locals {
   management_tags = {
     prime-app      = "simple-report"
     environment    = local.env
-    resource_group = "${local.project}-${local.name}-${local.env}"
```

Comment:
This was incorrectly tagged. `"${local.project}-${local.name}-${local.env}"` resolves to `prime-simple-report-dev3` when it should have been `prime-simple-report-dev`.